cmake/bob_robotics.cmake

```diff
@@ -94,13 +94,17 @@ macro(BoB_project)
             get_filename_component(GENN_USERPROJECT ${GENN_BIN_PATH}/../userproject/include ABSOLUTE)
             message("GeNN found in ${GENN_USERPROJECT}")
             BoB_add_include_directories(${GENN_USERPROJECT})
-
-            if(PARSED_ARGS_GENN_MODEL)
+            # If the user just wants the userprojects headers...
+            if(PARSED_ARGS_INCLUDE_GENN_USERPROJECTS)
                 # On *nix link dl
                 if(NOT WIN32)
                     BoB_add_link_libraries(dl)
                 endif()
 
+                # If GeNN is installed to /usr the above method for finding
+                # userprojects headers won't work
+                BoB_add_include_directories(/usr/include/genn)
```

Comment:
I'm guessing our /usr install (which sadly I seldom use/test) doesn't do anything sensible with the userprojects includes?

Reply:
Nope :disappointed:. For my Arch package I just copy over the ``userprojects`` folder and symlink the headers into /usr/include: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=genn

Reply:
next major version I'll tidy the mess of userprojects up...I kinda deprecated ``/userprojects/include`` and then resurrected it, forgetting that it wasn't really installable

Reply:
Do arch packages get rebuilt automatically with releases btw?

Reply:
I have to update the package and push it back to the AUR but if there is anyone out there who is also using the package then they should just get updates through their package manager (though it will build it from source).